ckanext/syndicate/controllers/syndicate.py

```diff
@@ -0,0 +1,107 @@
+import ckan.lib.base as base
+import ckan.lib.helpers as h
+from ckan.common import response, c, _
+import ckan.model as model
+import ckan.plugins.toolkit as tk
+from ckan.logic import NotAuthorized, NotFound
+from sqlalchemy.orm.exc import NoResultFound
+from ckanext.syndicate.plugin import syndicate_dataset
+
+import json
+
+abort = base.abort
+
+
+def _get_tasks(group_id):
+    tasks = []
+
+    try:
+        tasks = model.Session.query(model.TaskStatus).join(
+            model.Package, model.TaskStatus.entity_id == model.Package.id
+        ).join(
+            model.Group, model.Package.owner_org == model.Group.id
+        ).filter(model.Group.id == group_id).all()
+    except NoResultFound:
+        pass
+
+    return tasks
+
+
+def _get_task_and_delete(pkg_id):
+    task = model.Session.query(model.TaskStatus).filter(
+        model.TaskStatus.entity_id == pkg_id).one()
+    pkg_dict = {
+        'id': task.entity_id,
+        'state': task.state
+    }
+    task.delete()
+    model.Session.commit()
+
+    return pkg_dict
+
+
+def _delete_log_item(pkg_id):
+    delete_item = model.Session.query(model.TaskStatus).filter(
+        model.TaskStatus.entity_id == pkg_id).delete()
+    model.Session.commit()
+
+    return delete_item
+
+
+class SyndicateController(base.BaseController):
+    """Controller that provides syndicate admin UI."""
+
+    def admin_user_interface(self, id):
```

Comment:
Can you use for this action name, that will reflect its purpose? I don't know, `task_list` maybe?